### DISCUSSION TITLE
How the remember me option affects the SSO and client sessions lifespan

### DISCUSSION THREAD

**Discussion by fidgi** [keycloak/keycloak#14178] [Q&A] [answered]
When the user select the "remember me" option , does it apply globaly to the SSO session or only to the sessions of the clients for which the user has activated the "remember me" option ?

**Comment by darius-m** [comment, marked as answer]
I do not think users are prompted to select `Remember me` for each client. As far as I understand, the option sets a cookie in the user's browser that can be used for authentication in the `Browser` authentication flow (it is one of the alternatives at the top). As such, the option affects the SSO service (Keycloak) itself, not an individual client.

SSOs work by handling user authentication through browser redirects - i.e. the client requests user authentication, the SSO authenticates the user (using cookies, username + password, etc.) and then return a response to the client. The client itself is not a part of the authentication process, as it only sends requests and receives replies from the SSO server.

**Reply by fidgi** [reply]
thanks @darius-m .
My question was related to the fact that I see a growing number of long-live client sessions (i'm not talking about offline sessions but "standard" ones") in the infinispan cache and I was wondering whether it was related to a "remember me" option per client.
We're running Keycloak 17.0.1 Legacy version
